test/v2/unit/wallet.js

```diff
@@ -531,41 +531,49 @@ describe('V2 Wallet:', function() {
       .should.be.rejectedWith({ code: 'cpfpfeerate_not_nonnegative_integer' });
     }));
 
-    it('fails if maxFee is not passed and neither is noMaxFee', co(function *() {
-      yield wallet.accelerateTransaction({ cpfpTxIds: ['id'], noCpfpFeeRate: true })
```

Comment:
ahhh maybe you `noCpfpFeeRate: true` meant to say that we are not passing `cpfpFeeRate`? I thought it was a mistyped `noMaxFee`